packages/dagu_car_dbv2/script/test.py

```diff
@@ -0,0 +1,269 @@
+    def car_cmd_callback(self, msg_car_cmd):
+        if not self.actuator_limits_received:
+            self.pub_actuator_limits.publish(self.msg_actuator_limits)
+
+        # assuming same motor constants k for both motors
+        k_wheel = self.k                                                        #changed "k_r" to "k_wheel", because there is only one such konstant need now, RFMH_2019_02_25
+        k_l = self.k
+
+        # adjusting k by gain and trim
+        k_wheel_inv = (self.gain + self.trim) / k_wheel                             #changed "k_r" to "k_wheel", RFMH_2019_02_25
+        k_l_inv = (self.gain - self.trim) / k_l
+
+        omega_wheel = msg_car_cmd.v / self.radius                               #omega_r changed to omega_wheel and skipped the whole calculation
+        gamma = pow(pow((msg_car_cmd.v / msg_car_cmd.omega),2.0) - pow(self.cog_distance,2.0),-0.5) * self.axis_distance     #omega_l changed to gamma as this is the steering angle and inserted the new calculation: gamma = f(v, omega)
+
+        # conversion from motor rotation rate to duty cycle
+        # u_r = (gain + trim) (v + 0.5 * omega * b) / (r * k_r)
+        u_wheel = omega_wheel * k_wheel_inv                                             #omega_r changed to omega_wheel, u_r to u_wheel and k_r_inv to k_wheel_inv, RFMH_2019_02_25
+        # u_l = (gain - trim) (v - 0.5 * omega * b) / (r * k_l)
+        u_l = omega_l * k_l_inv
+
+        # limiting output to limit, which is 1.0 for the duckiebot
+        u_wheel_limited = max(min(u_wheel, self.limit), -self.limit)            #u_r_limited changed to "u_wheel_limited" and "u_r" changed to "u_wheel", RFMH_2019_02_25
+        u_l_limited = max(min(u_l, self.limit), -self.limit)
+
+        # Put the wheel commands in a message and publish
+        msg_wheels_cmd = WheelsCmdDBV2Stamped()
+        msg_wheels_cmd.header.stamp = msg_car_cmd.header.stamp
+        msg_wheels_cmd.vel_right = u_wheel_limited                              #vel_right is defined in the WheelsCmdDBV2Stamped --> name needs to be changed everywhere!, RFMH_2019_02_25
+        msg_wheels_cmd.vel_left = u_l_limited
+        self.pub_wheels_cmd.publish(msg_wheels_cmd)
+
+    def setup_parameter(self, param_name, default_value):
+        value = rospy.get_param(param_name, default_value)
+        # Write to parameter server for transparency
+        rospy.set_param(param_name, value)
+        rospy.loginfo("[%s] %s = %s " % (self.node_name, param_name, value))
+        return value
+
+if __name__ == '__main__':
+    rospy.init_node('inverse_kinematics_node', anonymous=False)
+    inverse_kinematics_node = InverseKinematicsNode()
+    rospy.spin()
```

Comment:
do these test work ? Otherwise I would not merge them until they are tested and fixed